tardis/plasma/standard_plasmas.py

```diff
@@ -175,15 +180,33 @@ def assemble_plasma(config, model, atom_data=None):
         property_kwargs[MarkovChainTransProbsCollector] = {
             "inputs": transition_probabilities_outputs
         }
-        if config.plasma.nlte_ionization_species:
-            nlte_ionization_species = config.plasma.nlte_ionization_species
-            for species in nlte_ionization_species:
-                if not (species in config.plasma.continuum_interaction.species):
-                    raise PlasmaConfigError(
-                        f"NLTE ionization species {species} not in continuum species."
-                    )
+        if (
+            config.plasma.nlte_ionization_species
+            or config.plasma.nlte_excitation_species
+        ):
+            nlte_ionization_species = []
+            nlte_excitation_species = []
```

Comment:
Why is this needed?

Reply:
so that if one of them is empty, it passes an empty list.

Reply:
I see. I think you could also achieve this by moving the assignment `nlte_ionization_species = config.plasma.nlte_ionization_species` and `nlte_excitation_species = config.plasma.nlte_excitation_species` here since the default value for both is an empty list (as defined in the yaml schema).